src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double leg;
+    private double base1;
+    private double base2;
```

Comment:
I can't understand what do these values mean
let's use either `height` and `middleLine`
or `firstBase`, `secondBase` and `height`